### PR TITLE
CCITCARBON-273 Create a Sidebar for GitHub Pages Docs

### PR DESCRIPTION
**Pull request by shay6**

### CODE REVIEW

**Comment by shay6** [comment]
See the example on my repo
https://shay6.github.io/teflo_webhooks_notification_plugin/

**Review by JunqiZhang0 — APPROVED** [review]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+---
+
 # Teflo Webhooks Notification Plugin
+{: .no_toc }
+
 
 This is Teflo's plugin for sending teflo notifications to chat application using webhooks.
 This plugin is used to send notifications on triggers like `on_task`, `on_start`, `on_failure`, `on_demand`
 during and post teflo execution. For more information on how to setup triggers for notification
 please visit [here](https://teflo.readthedocs.io/en/latest/users/definitions/notifications.html#triggers).
```
Comment (rujutashinde) [inline docs/index.md:14]: Could you also add the Teflo main page (the read the docs page ) , like you have added in other plugin docs

**Review by rujutashinde — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (shay6) [reply docs/index.md:14]: yes, I'll create a PR for teflo repo to modify [this](https://teflo.readthedocs.io/en/latest/users/install.html) page in readthedocs

Comment (rujutashinde) [reply docs/index.md:14]: No i dont mean to add anything on teflo, i wanted you to add the teflo's main page on this plugin's page
Please add this:

"To know more about teflo please visit here <teflo's read the docs link>"

Comment (shay6) [reply docs/index.md:14]: done
https://github.com/RedHatQE/teflo/pull/109

Comment (shay6) [reply docs/index.md:14]: Ok, I'll add this line. my bad

Comment (shay6) [reply docs/index.md:14]: done

**Review by rujutashinde — APPROVED** [review]

**Pull request merged**